tfjs-core/src/ops/browser.ts

```diff
@@ -154,6 +154,68 @@ function fromPixels_(
   return tensor3d(values, outShape, 'int32');
 }
 
+/**
+ * Creates a `tf.Tensor` from an image.
+ *
+ * ```js
+ * const image = new ImageData(1, 1);
+ * image.data[0] = 100;
+ * image.data[1] = 150;
+ * image.data[2] = 200;
+ * image.data[3] = 255;
+ *
+ * tf.browser.fromPixels(image).print();
```

Comment:
This is an async function. Maybe it should be used like:
`(await tf.browser.fromPixelsAsync(image)).print()`?